src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataLatestIT.java

```diff
@@ -127,6 +137,73 @@ public void testUseConnectionCtx() throws SQLException {
     }
   }
 
+  /**
+   * This tests the ability to have quotes inside a schema or database. This fixes a bug where
+   * double-quoted function arguments like schemas, databases, etc were returning empty resultsets.
+   *
+   * @throws SQLException
+   */
+  @Test
+  public void testDoubleQuotedDatabaseAndSchema() throws SQLException {
```

Comment:
Test that results are correctly returned for getColumns, getTables, and getSchemas when schema, table, and columns have double quotes in them.